jobs/apptypes_test.groovy

```diff
@@ -0,0 +1,63 @@
+evaluate(new File("${WORKSPACE}/common.groovy"))
+
+import utilities.StatusUpdater
+
+
+name = "apptypes_e2e"
+
+
+  job(name) {
+    description """
+      runs a e2e-runner job nightly we have to setup TEST=bps env variable for the docker container to execute the test
+    """.stripIndent().trim()
+
+    scm {
+      git {
+        remote {
+          github("deis/e2e-runner")
+        }
+        branch('master')
+      }
+    }
+
+    logRotator {
+      daysToKeep defaults.daysToKeep
+    }
+
+    parameters {
+      stringParam('RELEASE', defaults.workflow.release, "Release string for resolving workflow-[release](-e2e) charts")
```

Comment:
@smothiki should we change `defaults.workflow.release` to `dev`?  The former will pickup the current/most recent release chart while the latter will pick up, of course, the latest `dev` variant.